app/build.gradle

```diff
@@ -102,12 +108,17 @@ dependencies {
     implementation "com.android.support:recyclerview-v7:$supportVersion"
     implementation "com.android.support:design:$supportVersion"
 
+    // Architecture
+    implementation "android.arch.lifecycle:runtime:1.0.3"
+    annotationProcessor "android.arch.lifecycle:compiler:1.0.0"
```

Comment:
use `kapt` instead of `annotationProcessor`

Also, should these version numbers all match?